### PR TITLE
Submit coverage to codacy only if secret is available

### PR DESCRIPTION
**Pull request by CyberDem0n**
If PR is open from the external GH repo secrets are not set due to security reasons. It makes codacy coverage report to fail.

### CODE REVIEW

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -149,6 +149,8 @@ jobs:
       run: bash <(curl -Ls https://coverage.codacy.com/get.sh) report -r cobertura.xml -l Python --partial
       env:
         CODACY_PROJECT_TOKEN: ${{ secrets.CODACY_PROJECT_TOKEN }}
+        SECRETS_AVAILABLE: ${{ secrets.CODACY_PROJECT_TOKEN != '' }}
```
Comment (hughcapet) [inline .github/workflows/tests.yaml:152]: should we move them both to the workflow level 🤔

Comment (CyberDem0n) [reply .github/workflows/tests.yaml:152]: Please try

**Comment by CyberDem0n** [comment]
:+1:

**Comment by hughcapet** [comment]
👍

**Pull request merged**